### PR TITLE
python3Packages.pygogo: init at 0.13.2

### PR DESCRIPTION
**Pull request by drewrisinger**
###### Motivation for this change
Add a useful python logging library/application.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [x] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by drewrisinger** [comment]
@danieldk would you mind reviewing?

File: pkgs/development/python-modules/pygogo/default.nix [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  disabled = pythonOlder "3.6";
+
+  src = fetchFromGitHub {
+    owner = "reubano";
+    repo = "pygogo";
+    rev = "v${version}";
+    sha256 = "19rdf4sjrm5lp1vq1bki21a9lrkzz8sgrfwgjdkq4sgy90hn1jn9";
+  };
+
+  buildInputs = [ pkutils ];
```
Comment (danieldk) [inline pkgs/development/python-modules/pygogo/default.nix:24]:
```suggestion
  nativeBuildInputs = [ pkutils ];
```

**Review by danieldk — CHANGES_REQUESTED** [review]
`nix-review` PR fails with:

```
builder for '/nix/store/9rx0vab2k0pbj2al86kcfraxcv7nh95x-python2.7-pkutils-1.1.0.drv' failed with exit code 1; last 10 log lines:
      TypeError: Expected str or VersionInfo instance, but got <type 'unicode'>
  
  
  Name         Stmts   Miss  Cover
  --------------------------------
  pkutils.py      86     11    87%
  ----------------------------------------------------------------------
  Ran 12 tests in 0.020s
  
  FAILED (failures=3)
cannot build derivation '/nix/store/kd40g88p01m2g7kn339dirz54mhkvxi3-env.drv': 1 dependencies couldn't be built
```

**Review by jonringer — APPROVED** [review]
LGTM

Result of nixpkgs-review pr 91981 [1](https://github.com/Mic92/nixpkgs-review)<details>  <summary>4 packages built:</summary>  - python37Packages.pkutils  - python37Packages.pygogo  - python38Packages.pkutils  - python38Packages.pygogo</details>

**Pull request merged**